eng/pipelines/build.yml

```diff
@@ -55,6 +55,10 @@ stages:
               npm ci
             displayName: "Install packages"
             workingDirectory: $(Build.SourcesDirectory)/autorest.csharp
+          - script: |
+              npm run build
+            displayName: "Build Emitter.Csharp"
```

Comment:
A small question, what is the official name? CSharp or csharp or Csharp

Reply:
The folder name in source code is `Emitter.Csharp`. The official package name is `@azure-tools/cadl-csharp`